app.rb

```diff
@@ -71,11 +60,11 @@ def create_student
 
   def create_teacher
     puts 'Create a new teacher'
-    print 'Enter teacher age'
+    print 'Enter teacher age: '
     age = gets.chomp.to_i
-    print 'Enter teacher name'
+    print 'Enter teacher name: '
     name = gets.chomp
-    print 'Enter teacher specialization'
+    print 'Enter teacher specialization: '
     specialization = gets.chomp
```

Comment:
- [ ] in the `create_teacher` and `create_student` methods you are repeating some code like asking for the name and age this is not in line with the DRY rules, please make sure to avoid repeating code.